docs/src/Examples/Demo/DatePicker/BasicDatePicker.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { Fragment, PureComponent } from 'react';
-import { DatePicker } from 'material-ui-pickers';
+import DatePicker from 'material-ui-pickers/DatePicker';
```

Comment:
Can we do not show this usage in examples? esm bundle is relatively small and will be even smaller. Path imports can be used by advanced users.

Otherwise we will have the same confusion like in the latest issue.

Reply:
Nice catch. I will redo it back :)